style.css

```diff
@@ -46,14 +52,93 @@ body {
 
 /* ===== bootswatch ===== */
 .theme-darkly .form-control, .theme-darkly .form-control:focus {
-    background-color: #303030;
+    background-color: #303030 !important;
     color: #fff;
 }
 
+/*Changes the color of the grey differences to be a bit lighter in dark theme*/
+.theme-darkly .text-secondary{
+	color: #bdbdbd !important; 
+}
+
+/*Fixes the selected header and sort arrows in the light theme*/
+.theme-flatly .ng-table th.sortable.sort-asc,.theme-flatly .ng-table th.sortable.sort-desc{
+	background-color: #e7e8e9;
+	color: #000;
+}
+
+.theme-flatly .ng-table th.sortable.sort-desc .sort-indicator::after {
+    border-top-color: #000;
+}
+
+.theme-flatly .ng-table th.sortable .sort-indicator::after, .theme-flatly .ng-table th.sortable .sort-indicator::before {
+    border-color: #000 transparent;
+}
+
+
 /* ===== custom table width ===== */
 @media (min-width: 992px) {
-    .min-col-width { width: 1%; }
+    .min-col-width { width: 10%; }
```

Comment:
Yes, it's for the rerolled column but also the pop column since that one gets really thin and awkward
![image](https://user-images.githubusercontent.com/49487896/56895371-769aff80-6a78-11e9-8a7f-63f449aefdc6.png)

Reply:
I noticed that this also effects the other columns with data which in my opinion makes the table look way too crowded.
![image](https://user-images.githubusercontent.com/49487896/56895570-050f8100-6a79-11e9-9b09-1abac9fd0ad3.png)

Reply:
Aha, I see. Its more than just one column. Then yea, we will keep it at 10%.